src/components/Button/PlayButton.js

```diff
@@ -12,8 +14,7 @@ const hoverColorClasses = {
 }
 
 const centerButtonRules = [
-  'top: 50%;',
-  'bottom: auto;'
+  'top: 50%; margin-top: -2rem;'
```

Comment:
Why is the hard-coded margin needed? Can this be done with something more flexible/responsive?